QRCoderTests/ArtQRCodeRendererTests.cs

```diff
@@ -23,7 +23,7 @@ public void can_create_standard_qrcode_graphic()
             var bmp = new ArtQRCode(data).GetGraphic(10);
 
             var result = HelperFunctions.BitmapToHash(bmp);
-            result.ShouldBe("cb38c3156eaf13cdfba699bdafc3a84c");
+            result.ShouldBe("df510ce9feddc0dd8c23c54e700abbf0");
```

Comment:
Why did the hashes change? Can't see a change in the actual implementation of the renderer. Shouldn't they stay consistent?

Reply:
All calls to `BitmapToHash` use the new hashing algorithm, which uses LockBits to copy the actual pixel values, rather than relying on the PNG encoder, so that the hash is deterministic regardless of platform.